README.md

```diff
@@ -50,3 +50,28 @@ $> cmake .. -DCMAKE_INSTALL_PREFIX=<PREFIX> -DCMAKE_BUILD_TYPE=<BUILD_TYPE>
 $> make -j
 $> make install
 ```
+
+### Checkstyle
+This project uses [clang-tidy](https://clang.llvm.org/extra/clang-tidy/) to verify the code style. This tool is provided with
+the clang extra tools. To enable the code style checking, add `-DCMAKE_CXX_CLANG_TIDY=clang-tidy` to the cmake command.  
```

Comment:
why newline ?

Reply:
To make the file more readable. What do you suggest: to keep everything on a single line (246 characters)? I think it's really hard to maintain, and my IDE has a feature to automatically wrap long line (120 characters)

Reply:
ok